modules/microshift-manifests-removal-scenario.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * microshift//running_applications/microshift-deleting-resource-manifests.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-manifests-removal-scenario_{context}"]
+== Manifest removal scenario
+Use the following procedure in the data removal scenario to completely delete the resource defined in the kustomize manifests.
+
+.Procedure
```

Comment:
We cannot reuse this procedure for ostree.
It makes sense for rpm systems (with a caveat that both /etc/microshift/manifests.d/delete/ and /usr/lib/microshift/manifests.d/delete can be used, because they're both writable on rpm system).

On ostree systems: those needs to be packaged as RPMs that install to right dir straight away - there's no moving allowed because at runtime the /usr is read-only. But I don't think it's a big deal - if user already has manifests on ostree, they know how to create them, if they want to create "delete" manifests, they just need to use slightly different path.

Reply:
separate module created for OSTree